projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-interface/wpcom-admin-interface.php

```diff
@@ -416,3 +416,77 @@ function wpcom_is_duplicate_views_experiment_enabled() {
 		return $is_enabled;
 	}
 }
+
+/**
+ * Displays a notice when a user visits the enforced WP Admin view of a removed Calypso screen for
+ * the first time.
+ */
+function wpcom_show_removed_calypso_screen_notice() {
+	$current_screen = wpcom_admin_get_current_screen();
+
+	if ( ! in_array( $current_screen, WPCOM_DUPLICATED_VIEW, true ) ) {
+		return;
+	}
+
+	$dismissed_notices = get_user_option( 'wpcom_removed_calypso_screen_dismissed_notices' );
+	if ( is_array( $dismissed_notices ) && in_array( $current_screen, $dismissed_notices, true ) ) {
+		return;
+	}
+
+	if ( ! wpcom_is_duplicate_views_experiment_enabled() ) {
+		return;
+	}
+
+	remove_filter( 'pre_option_wpcom_admin_interface', 'wpcom_admin_interface_pre_get_option' );
+	$uses_wp_admin_interface = get_option( 'wpcom_admin_interface' ) === 'wp-admin';
+	add_filter( 'pre_option_wpcom_admin_interface', 'wpcom_admin_interface_pre_get_option', 10 );
+	if ( $uses_wp_admin_interface ) {
+		return;
+	}
+
+	remove_filter( 'get_user_option_jetpack_admin_menu_preferred_views', 'wpcom_admin_get_user_option_jetpack' );
+	$preferred_views = get_user_option( 'jetpack_admin_menu_preferred_views' );
+	add_filter( 'get_user_option_jetpack_admin_menu_preferred_views', 'wpcom_admin_get_user_option_jetpack' );
+	if ( ! empty( $preferred_views ) && isset( $preferred_views[ $current_screen ] ) && $preferred_views[ $current_screen ] === 'classic' ) {
+		return;
```

Comment:
Just out of curiosity, what happens if we don't remove the filers before getting the options?

Reply:
The notices would be displayed to users currently using the classic view (i.e. tangled wp-admin) or the classic admin interface (i.e. untangled wp-admin).